Classes/Controller/AbstractController.php

```diff
@@ -29,6 +29,6 @@ protected function getLanguageService(): LanguageService
      */
     protected function getCurrentDashboard(): string
     {
-        return $this->getBackendUser()->getModuleData('web_dashboard/current_dashboard/') ?: 'default';
+        return $this->getBackendUser()->getModuleData('web_dashboard/current_dashboard/') ?? '';
```

Comment:
right, a leftover of the refactoring, this should also solve the issue above. 
My idea was to have every time at least one dashboard, and if it not exists, it will be created.
I will push an update for this area.